backend/director/agents/transcript_agent.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from director.agents.base import BaseAgent, AgentResponse, AgentStatus
+from director.core.session import TextContent, MsgStatus
+from director.tools.videodb_tool import VideoDBTool
+
+logger = logging.getLogger(__name__)
+
+class VideoTranscriptionAgent(BaseAgent):
```

Comment:
Naming should be consistent.

1 - Existing agent file doesn't have _agent
`transcript_agent` -> `transcription.py`

2 - Agent name `VideoTranscriptionAgent` -> `TranscriptionAgent` (since in future we may reuse this for audio transcription as well)

3 - agent_name `video_transcription` -> `transcription_agent`

Reply:
@sarfarazsiddiquii agent name is still video_transcription can you make it simply `transcription`.

Reply:
This has been resolved in the recent commit. Sorry for overlooking it.